### PR TITLE
Add JSDoc 

### PR DESCRIPTION
**Pull request by OR13**
I've published a new version of the framework, which includes documentation.

These changes update to that version.

### CODE REVIEW

**Comment by OR13** [comment]
I want to get the docs publishing on github pages again.

File: docs/transmute-cli/1.0.0/index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    
+
+
+
+
+
+
+
+
+
```
Comment (eolszewski) [inline docs/transmute-cli/1.0.0/index.html]: These line carriages should be cleaned up.

File: docs/transmute-cli/1.0.0/index.js.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+<nav>
+    <h2><a href="index.html">Home</a></h2><h3>Modules</h3><ul><li><a href="module-TransmuteCLI.html">TransmuteCLI</a></li></ul>
+</nav>
+
+<div id="main">
+    
+    <h1 class="page-title">index.js</h1>
+    
+
+
```
Comment (eolszewski) [inline docs/transmute-cli/1.0.0/index.js.html]: whitespace

File: docs/transmute-cli/1.0.0/index.html (hunk 1 above)
Comment (eolszewski) [reply docs/transmute-cli/1.0.0/index.html]: oh, these are autogenerated... nvm...

File: docs/transmute-cli/1.0.0/index.js.html (hunk 2 above)
Comment (eolszewski) [reply docs/transmute-cli/1.0.0/index.js.html]: ignore

**Pull request merged**